### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by rgommers**

### CODE REVIEW

**Comment by rgommers** [comment]
Thanks @stefanv! The Japanese language preview works now @AtsushiSakai: https://deploy-preview-639--numpy-org.netlify.app/ja/

**Comment by stefanv** [comment]
I presume the dropdown language selector will be enabled at some point?

**Comment by rgommers** [comment]
> I presume the dropdown language selector will be enabled at some point?

Yes, that is planned to be done over the coming weeks. But there's work to do to validate translations and ensure they're complete enough, and then do a proper launch.

**Comment by steppi** [comment]
@rgommers is there a reason this is marked draft? It seems to be ready to merge.

**Comment by rgommers** [comment]
Copying my answer from Slack here: this branch/PR contains everything, and I don't think it was fully validated. I wanted to split off language by language a la gh-456. But there's no real reason to not merge this big branch if it helps. As long as it's a one-time or semi-frequent thing I guess. Because if that is merged, Crowdin will open a new PR as soon as one string gets translated. So I'm thinking we should accumulate them for a while, and then do a new translations release. I'm sure it's not too difficult to optimize the PR flow later. 

tl;dr do what you think is best.

**Comment by steppi** [comment]
Closing this and migrating translations to the Scientific Python Crowdin enterprise organization.
